src/view_neume.cpp

```diff
@@ -111,6 +111,9 @@ void View::DrawNeume(DeviceContext *dc, LayerElement *element, Layer *layer, Sta
     
     std::map<std::string, NeumeGroup> *neumeGroups = Neume::GetGroups();
     
+    // Generate intm attribute for all nc children except the first
+    // This is necessary to properly render the neume
+    neume->GenerateChildMelodic();

```

Comment:
If referring to the drawing of neume groups:

It's out of the norm in verovio to not search through the tree to draw each element individually, but I think for Neumes specifically this is the most efficient way to do it. If we were to handle drawing of neumes through each nc component, then we would have to search through the siblings and determine which position in the neume each component is and it would be more messy in my opinion. We can discuss this further, perhaps we can find a more elegant solution.